website/docs/cdktf/concepts/resources.mdx

```diff
@@ -593,9 +593,9 @@ import imports.aws.security_group.*;
                 vpc_id="vpcs",
                 egress=[
                     {
-                        "fromPort": 0,
-                        "toPort": 0,
-                        "ciderBlocks": ["0.0.0.0/0"],
+                        "from_port": 0,
+                        "to_port": 0,
+                        "cider_blocks": ["0.0.0.0/0"],
```

Comment:
I think it's safe to fix this via a suggestion instead of re-generating them 👍
```suggestion
                        "cidr_blocks": ["0.0.0.0/0"],
```